promscale/send-data/remote-write.md

```diff
@@ -0,0 +1,302 @@
+# Write data to Promscale using the remote-write API
+
+Promscale provides a remote-write endpoint for ingesting data. The endpoint is
+compatible with Prometheus. You can also push data to Promscale from a custom
+application.
+
+To configure Prometheus for remote writing to Promscale, see the section on
+[writing from Prometheus][write-prometheus]. To use the remote-write endpoint
+with a custom application, see the following sections.
+
+## Overview of remote-write endpoint
+
+The remote-write endpoint is located at `http://<PROMSCALE_URL>:<PORT>/write`.
+The URL and port depend on your configuration. For example, if you're running
+locally on the default port, the endpoint is located at
+`http://localhost:9201/write`.
+
+Only the HTTP POST method is supported.
+
+The default data format is Protobuf, but JSON streaming and text are also
+supported.
+
+For more information on endpoint details, see the [Prometheus
+documentation][prometheus-remote-storage].
+
+## Formats
+
+This section covers formats for sending data to the Promscale endpoint,
+including:
+
+*   [General format for time-series data][general-format]
+*   [Protobuf format][protobuf-format]
+*   [JSON streaming format][json-format]
+*   [Prometheus and OpenMetric text format][text-format]
+
+## General format for time-series data
+
+A time series must contain both labels and samples.
+
+Labels are a collection of label name/value pairs with unique label names. Every
+request must contain a label named `__name__` which cannot have an empty value.
+All other labels are optional metadata.
+
+A sample must contain two things:
+
+*   An integer timestamp in milliseconds since the UNIX epoch. In other words,
+    the number of milliseconds since `1970-01-01 00:00:00 UTC`, excluding leap
+    seconds. This should be represented as required by Go's [ParseInt][parseint]
+    function.
+*   A floating point number that represents the actual measured value.
+
+## Protobuf format
+
+Protobuf is the default data format. It is slightly more complex than the [JSON
+streaming format][json-format], but it is usually more performant, because the
```

Comment:
```suggestion
Protobuf is the default data format. It is slightly more complex than the 
[JSON streaming format][json-format], but it is usually more performant, because the
```